test/components/previews/selection_preview/within_a_table.html.erb

```diff
@@ -0,0 +1,19 @@
+<table data-controller="selection">
+  <tbody>
```

Comment:
**nitpick (blocking):** Same comment here, we do not have a table component, but when I looked at the preview I though it was still a list, can you make it look more like a table by using some of Flowbite styles: https://flowbite.com/docs/components/tables/#default-table

Reply:
I thought the selection controller should be the :star: of the preview. Please see e019021985ff32e2b8378ecd2a0fa724db063c2c.

Reply:
Thanks, much better